sklearn/ensemble/forest.py

```diff
@@ -157,6 +158,10 @@ def __init__(self,
         self.warm_start = warm_start
         self.class_weight = class_weight
 
+        if self.n_estimators == 10:
+            warnings.warn("'n_estimators' default value was changed to 100 in version 0.20.", DeprecationWarning)
```

Comment:
What we want is "n_estimators default value will change to 100 in version 0.20".

By convention, this warning should only be issued in `fit`, not `__init__`.